### PR TITLE
Add Kiibohd Configurator v0.3.0

### PR DESCRIPTION
**Pull request by debo**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by commitay** [comment]
This should probably be in [`caskroom/drivers`](https://github.com/caskroom/homebrew-drivers)

> Client Side Configuration & Flashing Software for Kiibohd compatible keyboards.
___
```
macOS

Install dfu-util (available via homebrew)
```

Should this have a `depends_on formula: 'dfu-util'`?

**Comment by debo** [comment]
@commitay not sure, it's actually a GUI configuration tool rather than a driver. The keyboard itself is driverless. What do you think?

As per the depends on, definitely, I was looking for that but a google search led me to an issue that was close for lack of interest. I'll add the caveats.

**Comment by commitay** [comment]
Our general rule is:

> If the software is useless without the peripheral, it’s considered a driver.

**Comment by debo** [comment]
@commitay makes perfect sense, I'll move the PR then.

File: Casks/kiibohd-configurator.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  sha256 'bacc8c5e735d1cd851aebcba6b92bc772a1332a572361a6644b59808f012d07d'
+
+  # github.com/kiibohd/configurator was verified as official when first introduced to the cask
+  url "https://github.com/kiibohd/configurator/releases/download/v#{version}/kiibohd-configurator-#{version}-mac.dmg"
+  name 'The Kiibohd Configurator'
+  homepage 'http://kiibohd.com/'
+
+  app 'Kiibohd Configurator.app'
+
+  depends_on formula: 'dfu-util'
```
Comment (commitay) [inline Casks/kiibohd-configurator.rb]: This should be above the `app` stanza.

**Comment by debo** [comment]
@commitay PR moved [here](https://github.com/caskroom/homebrew-drivers/pull/233) also addressing your last comment.

**Comment by commitay** [comment]
Thanks!

**Comment by debo** [comment]
No problem. Thanks to you for the valuable inputs.

**Pull request closed without merging**